jgrapht-ext/src/test/java/org/jgrapht/ext/DOTImporterTest.java

```diff
@@ -47,9 +53,9 @@ public void testUndirectedWithLabels() throws ImportException {
 
       Multigraph<String, DefaultEdge> expected
             = new Multigraph<String, DefaultEdge>(DefaultEdge.class);
-      expected.addVertex("abc123");
-      expected.addVertex("fred");
-      expected.addEdge("abc123", "fred");
+      expected.addVertex("1");
+      expected.addVertex("2");
+      expected.addEdge("1", "2");
 
```

Comment:
@jkinable, fred and abc123 are the labels and not the vertex identifiers.

The input is 
```
graph G {
  1 [ label="abc123"];
  2 [ label="fred"];
  1 -- 2; 
}
```

which means that there are two vertices 1 and 2 and one edge between them. The labels are just extra attributes and no one forces them to be distinct. Moreover, they can also be absent. The following is also valid.

```
graph G {
  1 -- 2; 
}
```

as well as 

```
graph G {
  1 [ label="same"];
  2 [ label="same"];
  1 -- 2; 
}
```

The bug was that if labels existed, they were used as identifiers.